test/test_mpas.py

```diff
@@ -110,7 +110,7 @@ def test_set_attrs(self):
         ]
 
         # included attrs: 'sphere_radius', 'mesh_spec' 'on_a_sphere'
-        ds = _read_mpas(self.mpas_xr_ds)
+        ds, _ = _read_mpas(self.mpas_xr_ds)
```

Comment:
every read function for our I/O now returns a ugrid data mapping in addition to the dataset, since this was for testing we did not need the ugrid mapping

Reply:
Is this `mapping` test anywhere?

Reply:
Currently it's not, however right now it's mostly acting as a placeholder for eventual use. Will write tests when we fully populate it.